### PR TITLE
updated tests with helper classes

### PR DESCRIPTION
**Pull request by nphias**
fixes bug : https://github.com/evomimic/map-holons/issues/100

### CODE REVIEW

File: tests/src/map_holons/holons/common.ts
Comment (evomimic) [inline tests/src/map_holons/holons/common.ts:1]: Keep this clean by just making parameters cell and request:DanceRequestObject

Comment (nphias) [reply tests/src/map_holons/holons/common.ts:1]: code was cleaned up and ready to merge with main

**Comment by evomimic** [comment]
- The get_all_holons dance retrieves all holons from the persistent store (i.e., DHT) NOT the staging area.
- I'm proposing we stick with the terminology _stage_new_holon_ in preference to _create_ or _add_ to reinforce the distinction between staging holons and committing them to the persistent store.

I added a commit that updated the test case console messages to more accurately reflect the above.

**Comment by evomimic** [comment]
What this enhancement brought into focus is the need for a deeper discussion and definition of the MAP's client-side architecture. Essentially, we need to implement the _model layer_ of the client-side MVC architecture for the MAP's Uniform API. Since the Uniform API is THE interface for invoking everything in the MAP, it is worth devoting some extra attention to it.

I believe this _model layer_ will look significantly different than the _tests/src/map_holons/holons/common.ts_ definitions included in this PR. But rather than hold up this PR, I'm going to proceed with the merge and document my thoughts elsewhere regarding design of the _model layer_ classes and functions.

**Pull request merged**